src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -88,9 +111,19 @@ export function PharmacyCard(props: PharmacyProps) {
                   <Link url={props.website}>{props.website}</Link>
                 </Stack.Item>
               </Stack>
+              <Stack>
+                {showMap ? null : (
+                  <Stack.Item>
+                    <Button icon={LocationMajor} onClick={showTheMap}>
```

Comment:
Why are you using an arrow here? To manage context binding? You don;'t need to worry about that in functional components.

Reply:
![image](https://user-images.githubusercontent.com/19317416/117183947-290fa680-ada6-11eb-9aac-8cf8207b6f27.png)
Not super familiar with React, is there a workaround for this error when not using an arrow function?

Reply:
Sorry, I should have been more clear:

please go ahead and use `onClick={() =>  { setShouldShowMap(true) }}`

I just didn't understand my you were calling `setShouldShowMap` in one place and then calling an arrow function const in a click event handler. The const arrow function was a useful pattern people used for class based components to manage context bindings, so I think I ended up just being confused :)